### ISSUE TITLE
Install of VS2017 Build January 27, 2017 (build 26127.00) Fails

### ISSUE THREAD

**Issue by MarkMichaelis** [dotnet/roslyn-project-system#1362]
I am trying to install VS201 but it fails with the log mentioned below.  This occurs whether or not I have any "install workloads" or not.

```
The product failed to install the listed workloads and components due to one or more package failures.

Incomplete workloads
    .NET desktop development (Microsoft.VisualStudio.Workload.ManagedDesktop,version=15.0.26127.0)
    ASP.NET and web development (Microsoft.VisualStudio.Workload.NetWeb,version=15.0.26127.0)
    Visual Studio core editor (Microsoft.VisualStudio.Workload.CoreEditor,version=15.0.26004.1)

Incomplete components
    .NET desktop development tools (Microsoft.VisualStudio.Component.ManagedDesktop.Prerequisites,version=15.0.26109.1)
    .NET Framework 4.6.1 development tools (Microsoft.Net.ComponentGroup.DevelopmentPrerequisites,version=15.0.26004.1)
    .NET Portable Library targeting pack (Microsoft.VisualStudio.Component.PortableLibrary,version=15.0.26109.1)
    ASP.NET and web development tools (Microsoft.VisualStudio.Component.Web,version=15.0.26127.0)
    Blend for Visual Studio (Microsoft.ComponentGroup.Blend,version=15.0.26004.1)
    C# and Visual Basic (Microsoft.VisualStudio.Component.Roslyn.LanguageServices,version=15.0.26109.1)
    ClickOnce Publishing (Microsoft.Component.ClickOnce,version=15.0.26004.1)
    Cloud Explorer (Microsoft.VisualStudio.Component.CloudExplorer,version=15.0.26109.1)
    Common Azure tools (Microsoft.VisualStudio.Component.Common.Azure.Tools,version=1.9)
    Data sources and service references (Microsoft.VisualStudio.Component.VisualStudioData,version=15.0.26004.1)
    Data sources for SQL Server support (Microsoft.VisualStudio.Component.SQL.DataSources,version=15.0.26004.1)
    Developer Analytics tools (Microsoft.VisualStudio.Component.AppInsights.Tools,version=15.0.26127.0)
    Entity Framework 6 tools (Microsoft.VisualStudio.Component.EntityFramework,version=15.0.26004.1)
    IIS Express  (Microsoft.VisualStudio.Component.IISExpress,version=15.0.26004.1)
    IntelliTrace (Microsoft.VisualStudio.Component.IntelliTrace.FrontEnd,version=15.0.26004.1)
    JavaScript and TypeScript language support (Microsoft.VisualStudio.Component.JavaScript.TypeScript,version=15.0.26004.1)
    Live Unit Testing (Microsoft.VisualStudio.Component.LiveUnitTesting,version=15.0.26109.1)
    Managed Desktop Workload Core (Microsoft.VisualStudio.Component.ManagedDesktop.Core,version=15.0.26109.1)
    NuGet package manager (Microsoft.VisualStudio.Component.NuGet,version=15.0.26004.1)
    Profiling tools (Microsoft.VisualStudio.Component.DiagnosticTools,version=15.0.26109.1)
    SQL Server Data Tools (Microsoft.VisualStudio.Component.SQL.SSDT,version=15.0.26109.1)
    Visual Studio core editor (Microsoft.VisualStudio.Component.CoreEditor,version=15.0.26004.1)
    Web Deploy (Microsoft.VisualStudio.Component.WebDeploy,version=15.0.26004.1)
    Windows Communication Foundation (Microsoft.VisualStudio.Component.Wcf.Tooling,version=15.0.26109.1)

You can search for solutions using the information below, modify your selections for the above workloads and components and retry the installation, or remove the product from your machine.

Following is a collection of individual package failures that led to the incomplete workloads and components above. To search for existing reports of these specific problems, please copy and paste the URL from each package failure into a web browser. If the issue has already been reported, you can find solutions or workarounds there. If the issue has not been reported, you can create a new issue where other people will be able to find solutions or workarounds.

Package 'Microsoft.VisualStudio.TeamFoundation.TeamExplorer,version=15.112.26110.0' failed to install.
    Search URL: https://aka.ms/VSSetupErrorReports?q=PackageId=Microsoft.VisualStudio.TeamFoundation.TeamExplorer;PackageAction=Install;ReturnCode=3001
    Impacted workloads
        .NET desktop development (Microsoft.VisualStudio.Workload.ManagedDesktop,version=15.0.26127.0)
        ASP.NET and web development (Microsoft.VisualStudio.Workload.NetWeb,version=15.0.26127.0)
        Visual Studio core editor (Microsoft.VisualStudio.Workload.CoreEditor,version=15.0.26004.1)
    Impacted components
        .NET desktop development tools (Microsoft.VisualStudio.Component.ManagedDesktop.Prerequisites,version=15.0.26109.1)
        .NET Framework 4.6.1 development tools (Microsoft.Net.ComponentGroup.DevelopmentPrerequisites,version=15.0.26004.1)
        .NET Portable Library targeting pack (Microsoft.VisualStudio.Component.PortableLibrary,version=15.0.26109.1)
        ASP.NET and web development tools (Microsoft.VisualStudio.Component.Web,version=15.0.26127.0)
        Blend for Visual Studio (Microsoft.ComponentGroup.Blend,version=15.0.26004.1)
        C# and Visual Basic (Microsoft.VisualStudio.Component.Roslyn.LanguageServices,version=15.0.26109.1)
        ClickOnce Publishing (Microsoft.Component.ClickOnce,version=15.0.26004.1)
        Cloud Explorer (Microsoft.VisualStudio.Component.CloudExplorer,version=15.0.26109.1)
        Common Azure tools (Microsoft.VisualStudio.Component.Common.Azure.Tools,version=1.9)
        Data sources and service references (Microsoft.VisualStudio.Component.VisualStudioData,version=15.0.26004.1)
        Data sources for SQL Server support (Microsoft.VisualStudio.Component.SQL.DataSources,version=15.0.26004.1)
        Developer Analytics tools (Microsoft.VisualStudio.Component.AppInsights.Tools,version=15.0.26127.0)
        Entity Framework 6 tools (Microsoft.VisualStudio.Component.EntityFramework,version=15.0.26004.1)
        IIS Express  (Microsoft.VisualStudio.Component.IISExpress,version=15.0.26004.1)
        IntelliTrace (Microsoft.VisualStudio.Component.IntelliTrace.FrontEnd,version=15.0.26004.1)
        JavaScript and TypeScript language support (Microsoft.VisualStudio.Component.JavaScript.TypeScript,version=15.0.26004.1)
        Live Unit Testing (Microsoft.VisualStudio.Component.LiveUnitTesting,version=15.0.26109.1)
        Managed Desktop Workload Core (Microsoft.VisualStudio.Component.ManagedDesktop.Core,version=15.0.26109.1)
        NuGet package manager (Microsoft.VisualStudio.Component.NuGet,version=15.0.26004.1)
        Profiling tools (Microsoft.VisualStudio.Component.DiagnosticTools,version=15.0.26109.1)
        SQL Server Data Tools (Microsoft.VisualStudio.Component.SQL.SSDT,version=15.0.26109.1)
        Visual Studio core editor (Microsoft.VisualStudio.Component.CoreEditor,version=15.0.26004.1)
        Web Deploy (Microsoft.VisualStudio.Component.WebDeploy,version=15.0.26004.1)
        Windows Communication Foundation (Microsoft.VisualStudio.Component.Wcf.Tooling,version=15.0.26109.1)
    Log
        C:\Users\mark\AppData\Local\Temp\dd_setup_20170128152453_018_Microsoft.VisualStudio.TeamFoundation.TeamExplorer.log
    Details
        Command executed: "C:\Program Files (x86)\Microsoft Visual Studio\Installer\resources\app\ServiceHub\Services\Microsoft.VisualStudio.Setup.Service\vsixinstaller.exe" /q /s /admin /appidinstallpath:"C:\Program Files (x86)\Microsoft Visual Studio\2017\EnterpriseTake4" /logFile:"C:\Users\mark\AppData\Local\Temp\dd_setup_20170128152453_018_Microsoft.VisualStudio.TeamFoundation.TeamExplorer.log" /skuName:Enterprise /skuVersion:15.0.26113.0 /appidname:displayName "C:\ProgramData\Microsoft\VisualStudio\Packages\Microsoft.VisualStudio.TeamFoundation.TeamExplorer,version=15.112.26110.0\TeamExplorer.vsix"
        Return code: 3001
        Return code details: This VSIX extension failed.

```

@morrisjoe

**Comment by MarkMichaelis** [comment]
Note that I have tried running the [Total Uninsatller](https://github.com/Microsoft/VisualStudioUninstaller/releases) but the install still fails.
I have also changed the default location where VS is installed from `C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise` to `C:\Program Files (x86)\Microsoft Visual Studio\2017\EnterpriseTake2` but there is not change.  I am currently unable to install VS2017RC.
